export.py

```diff
@@ -4,47 +4,59 @@
 # https://github.com/smashwilson/slack-emojinator
 
 from __future__ import print_function
-from slacker import Slacker
+
+import requests
+import lxml.html
 
 import argparse
 import os
-import requests
 import shutil
 
+from upload import _session
+
 
 def _argparse():
     parser = argparse.ArgumentParser(
-        description='Bulk import of emoji froma slack team'
+        description='Bulk import of emoji from a slack team'
+    )
+    parser.add_argument(
+        'directory',
+        help='Where do we store downloaded emoji?'
     )
     parser.add_argument(
-        '--directory', '-d',
-        default=os.getenv('EMOJI_DIR'),
-        help='Defaults to the $EMOJI_DIR environment variable.'
+        '--team-name', '-t',
+        default=os.getenv('SLACK_TEAM'),
+        help='Defaults to the $SLACK_TEAM environment variable.'
     )
     parser.add_argument(
-        '--slack-api-token', '-s',
-        default=os.getenv('SLACK_API_TOKEN'),
-        help='Defaults to the $SLACK_API_TOKEN environment variable.'
+        '--cookie', '-c',
+        default=os.getenv('SLACK_COOKIE'),
+        help='Default sto the $SLACK_COOKIE environment variable.'
     )
     args = parser.parse_args()
     return args
 
+
 def main():
     args = _argparse()
-    download_emoji(args.directory, args.slack_api_token)
-
-def download_emoji(directory, slack_api_token):
-    print("slack API token: %s" % slack_api_token)
-    slack = Slacker(slack_api_token)
-    if not os.path.exists(directory):
-        os.makedirs(directory)
-    emojis = slack.emoji.list()
-    for emoji_name, emoji_url in emojis.body['emoji'].items():
-        if "alias" not in emoji_url:
+
+    if not os.path.exists(args.directory):
+        os.makedirs(args.directory)
+
+    session = _session(args)
+    resp = session.get(session.url)
+    tree = lxml.html.fromstring(resp.text)
+    urls = tree.xpath(r'//td[@headers="custom_emoji_image"]/span/@data-original')
+    names = [u.split('/')[-2] for u in urls]
+
+    for emoji_name, emoji_url in zip(names, urls):
+        if "alias" not in emoji_url:  # this does not seem necessary ...
```

Comment:
Looks like Slack allows custom aliases for standard emoji; is that what this conditional covers? If that's not the case then I'd just drop it.